<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Duplicates</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1185</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <widget class="QToolButton" name="go">
      <property name="text">
       <string>Run Now</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/go.png</normaloff>:/resources/go.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="dirName"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="selectDir">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="application.qrc">
        <normaloff>:/resources/fileopen.png</normaloff>:/resources/fileopen.png</iconset>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTreeView" name="files"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Ignored Directories:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QToolButton" name="addDir">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="application.qrc">
           <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="delDir">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="application.qrc">
           <normaloff>:/resources/delete.png</normaloff>:/resources/delete.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QListWidget" name="ignoredDirs"/>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>109</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Directory</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="ignoreHidden">
      <property name="text">
       <string>Ignore Hidden Files/Directories? (Including files starting with a .)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="iconText">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>dirName</tabstop>
  <tabstop>selectDir</tabstop>
  <tabstop>go</tabstop>
 </tabstops>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>CMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
